test/kvao/delete.suite.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const bdd = require('../helpers/bdd-if');
+const helpers = require('./_helpers');
+const should = require('should');
+
+module.exports = function(dataSourceFactory, connectorCapabilities) {
+  describe('delete', function() {
```

Comment:
This test suite should be disabled unless the connector supports the newly added feature, see https://github.com/strongloop/loopback-datasource-juggler/pull/1211#discussion_r94783632